spec/ndx-beadl.extensions.yaml

```diff
@@ -139,6 +139,11 @@ groups:
     neurodata_type_inc: VectorData
     dtype: float32
     doc: The time that the event occurred, in seconds.
+  - name: duration
+    neurodata_type_inc: VectorData
+    dtype: float32
+    doc: Duration of the event in seconds.
+    quantity: '?'
```

Comment:
Please make these changes in https://github.com/rly/ndx-beadl/blob/main/src/spec/create_extension_spec.py and run the script to update the YAML, rather than updating the YAML by hand. 

In addition, we should add an attribute `unit` on the duration column, that stores the value `seconds`.

Reply:
Should the timestamp column also have an attribute for units? @oruebel

Reply:
@rly should we add `unit` attributes for `duration` and `timestamp`? What does `ndx-events` do here?

Reply:
Yes, we should add an `unit` attributes for duration and timestamp like seen here: https://github.com/NeurodataWithoutBorders/nwb-schema/blob/dev/core/nwb.base.yaml#L179-L182

In general, it is against best practices to add attributes to a `VectorData` or other data type without defining a new type. But in this case, since the `unit` value will be fixed to "seconds", I think it is OK.

These attributes are not yet on ndx-events but I will add them there now.

Reply:
I can also take over from here in this PR, since the EventsTable in ndx-events should be identical to the EventsTable defined here.

Reply:
@rly Sounds good.

Reply:
> In general, it is against best practices to add attributes to a `VectorData` or other data type without defining a new type.

I think it would be reasonable to define a time `DurationVectorData` and `TimestampVectorData` for this, which would also make it easier to reuse these in other tables.

Reply:
I'll make these changes in a separate PR.